app/src/main/java/org/stepik/android/presentation/course/CoursePresenter.kt

```diff
@@ -125,7 +128,15 @@ constructor(
             .subscribeOn(backgroundScheduler)
             .subscribeBy(
                 onComplete = { state = CourseView.State.EmptyCourse },
-                onSuccess  = { state = CourseView.State.CourseLoaded(it); postCourseViewedNotification(it.courseId) },
+                onSuccess  = {
```

Comment:
Will it send event on every swipe to refresh?

Reply:
Moved logging to `ScreenManagerImpl`